--- Crypt-RIPEMD160-0.04/rmd160.h.orig	1998-04-07 11:32:36.000000000 +0200
+++ Crypt-RIPEMD160-0.04/rmd160.h	2004-08-28 22:45:08.162936140 +0200
@@ -20,12 +20,13 @@
 
 /********************************************************************/
 
+#include <stdint.h>
 /* typedef 8, 16 and 32 bit types, resp.  */
 /* adapt these, if necessary, 
    for your operating system and compiler */
-typedef    unsigned char        byte;   /* unsigned 8-bit integer */
-typedef    unsigned short       word;   /* unsigned 16-bit integer */
-typedef    unsigned long        dword;  /* unsigned 32-bit integer */ 
+typedef    uint8_t        byte;   /* unsigned 8-bit integer */
+typedef    uint16_t       word;   /* unsigned 16-bit integer */
+typedef    uint32_t       dword;  /* unsigned 32-bit integer */ 
 
 
 /********************************************************************/
